ord_schema/orm/scripts/add_datasets.py

```diff
@@ -41,28 +42,41 @@
 
 from ord_schema.logging import get_logger
 from ord_schema.message_helpers import load_message
-from ord_schema.orm.database import add_dataset, add_rdkit, delete_dataset, get_connection_string
+from ord_schema.orm.database import add_dataset, add_rdkit, delete_dataset, get_connection_string, get_dataset_md5
 from ord_schema.proto import dataset_pb2
 
 logger = get_logger(__name__)
 
 
-def _add_dataset(filename: str, url: str, overwrite: bool) -> None:
+def _add_dataset(filename: str, url: str, update: bool) -> None:
```

Comment:
I think the `overwrite` kwarg is more descriptive, but that is a minor nitpick

Reply:
That's fine; will change back.